doc/lsst.ap.verify/running.rst

```diff
@@ -0,0 +1,80 @@
+
+Datasets are identified by a name that gets mapped to an :ref:`eups-registered directory <ap-verify-datasets-install>` containing the data.
+The mapping is :ref:`configurable<ap-verify-configuration-dataset>`.
+
+.. _ap-verify-run-output:
+
+How to Run ap_verify in a New Workspace
+---------------------------------------
+
+Using the :ref:`HiTS 2015 <ap_verify_hits2015-package>` dataset as an example, one can run ``ap_verify`` as follows:
+
+.. TODO: why dataIdString instead of id or dataId? (DM-12853)
+
+.. code-block:: sh
+
+   $ python ap_verify/bin/ap_verify.py --dataset HiTS2015 --output workspace/hits/ --dataIdString "visit=54123 ccd=25 filter=g" --silent
+
+Here:
+
+* ``HiTS2015`` is the :ref:`dataset name <ap-verify-dataset-name>`,
+* ``workspace/hits/`` is the location of the :ref:`Butler repository<command-line-task-data-repo-using-uris>` in which the pipeline will work,
+* ``visit=54123 ccd=25 filter=g`` is the :ref:`dataId<command-line-task-dataid-howto-about-dataid-keys>` to process, and
+* ``--silent`` disables SQuaSH metrics reporting.
+
+This will create a workspace (a Butler repository) in ``workspace/hits`` based on ``<hits-data>/data/``, ingest the HiTS data into it, then run visit 54123 through the entire AP pipeline.
+
+.. note::
+
+   The command-line interface for ``ap_verify`` is at present much more limited than those of command-line tasks.
+   In particular, only file-based repositories are supported, and compound dataIds cannot be provided.
+   See the :ref:`ap-verify-cmd` for details.
+
+.. _ap-verify-run-rerun:
+
+How to Run ap_verify in the Dataset Directory
+---------------------------------------------
+
+It is also possible to place a workspace in a subdirectory of a dataset directory. The syntax for this mode is:
+
+.. code-block:: sh
+
+   $ python python/lsst/ap/verify/ap_verify.py --dataset HiTS2015 --rerun run1 --dataIdString "visit=54123 ccd=25 filter=g" --silent
+
+The ``--rerun run1`` argument will create a workspace in ``<hits-data>/rerun/run1/``.
+Since datasets are :ref:`not, in general, repositories<ap-verify-datasets-butler>`, the ``--rerun`` argument only superficially resembles the analogous argument for command-line tasks.
+In particular, ``ap_verify``'s ``--rerun`` argument does not support repository chaining (as in ``--rerun input:output``); the input for ``ap_verify`` will always be determined by the ``--dataset``.
+
+.. _ap-verify-results:
+
+How to Use Measurements of Metrics
+----------------------------------
+
+After ``ap_verify`` has run, it will produce a file named ``ap_verify.verify.json`` in the working directory.
+This file contains metric measurements in ``lsst.verify`` format, and can be loaded and read as described in the :ref:`verify documentation<lsst.verify>` or in `SQR-019 <https://sqr-019.lsst.io>`_.
```

Comment:
``lsst.verify documentation`` (for clarify, refer to other packages by their full namespace)